go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -3225,36 +3225,67 @@ var LogTableFunctionScriptTests = []queries.ScriptTest{
 		},
 		Assertions: []queries.ScriptTestAssertion{
 			{
-				Query:    "SELECT count(*) from dolt_log();",
-				Expected: []sql.Row{{4}},
+				Query: "SELECT message from dolt_log();",
+				Expected: []sql.Row{
+					{"inserting into t"},
+					{"creating table t"},
+					{"Initialize data repository"},
+				},
 			},
 			{
-				Query:    "SELECT count(*) from dolt_log('main');",
-				Expected: []sql.Row{{4}},
+				Query: "SELECT message from dolt_log('main');",
+				Expected: []sql.Row{
+					{"inserting into t"},
+					{"creating table t"},
+					{"Initialize data repository"},
+				},
 			},
 			{
-				Query:    "SELECT count(*) from dolt_log(@Commit1);",
-				Expected: []sql.Row{{3}},
+				Query: "SELECT message from dolt_log(@Commit1);",
+				Expected: []sql.Row{
+					{"creating table t"},
+					{"Initialize data repository"},
+				},
 			},
 			{
-				Query:    "SELECT count(*) from dolt_log(@Commit2);",
-				Expected: []sql.Row{{4}},
+				Query: "SELECT message from dolt_log(@Commit2);",
+				Expected: []sql.Row{
+					{"inserting into t"},
+					{"creating table t"},
+					{"Initialize data repository"},
```

Comment:
why are the counts changing? These are for existing tests, so I'd expect them to stay the same

Reply:
I updated the test harness to setting a setup commit since we're only doing reads in this test. Typically, there would be a checkpoint commit set with message "checkpoint enginetest database <db>" so that tests can reset to this point between query assertions. This would also show up on dolt log calls so I think it's less confusing to just skip setting it.

Reply:
L2049 in `dolt_engine_test.go`